mod/web_services/lib/web_services.php

```diff
@@ -357,7 +357,7 @@ function api_auth_key() {
 	}
 
 	// check that it is active
-	$api_user = get_api_user($CONFIG->site_id, $api_key);
+	$api_user = get_api_user($CONFIG->site_guid, $api_key);
```

Comment:
just do a search for $CONFIG in the mod directory and you will have some extra work